.github/workflows/continuous-integration-workflow.yml

```diff
@@ -17,24 +17,29 @@ jobs:
         config:
           - {dockerfile: 'fedora',   tag: 'fedora'}
           - {dockerfile: 'fedora',   tag: 'fedora_mpich',        build_args: 'MPI=mpich'}
-          - {dockerfile: 'fedora',   tag: 'fedora_rawhide',      build_args: 'TAG=rawhide,PYTHON=python3.11', continue-on-error: 'true'}
+          - {dockerfile: 'fedora',   tag: 'fedora_rawhide',      build_args: "TAG=rawhide\nPYTHON=python3.11", continue-on-error: 'true'}
```

Comment:
@XzzX a bit hackish, but it works.

Reply:
Is this correct: Github does not allow lists under `config`, however, checkout uses lists for `build_args`?

Reply:
If so, I am fine with the hack. :)

Reply:
The action doesn’t want a list but string block, e.g.
```
build-args: |
  VAR1=x
  VAR2=y
```

in one of my main attempts I tried an actual list
 ```
build-args:
  - VAR1=x
  - VAR2=y
```

but that failed